.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+**Description**
+
+This PR fixes #
+
+**Notes for Reviewers**
+
+<!--
+Thank you for contributing to Nighthawk! 
+
+Contributing Conventions:
+
+1. Build and test your changes before submitting a PR. 
```

Comment:
Maybe mention familiarizing oneself with https://github.com/envoyproxy/nighthawk/blob/master/CONTRIBUTING.md ?

Reply:
I guess you are right, also we can mention the DCO requirement I guess.

Reply:
Good idea, yes. There's also pre-push/pre-commit hooks in https://github.com/envoyproxy/nighthawk/tree/master/support that may help with that.

Reply:
I'd actually argue further to Otto's point that I'd prefer we not duplicate where our contributing conventions are.  I'd prefer to only reference the other markdown file here rather than specify some conventions and reference it here.

I could probably be convinced in a different direction if you feel strongly.

Reply:
Agree, we can reference the markdown file here than the other way around.